config/policies.js

```diff
@@ -30,6 +30,8 @@ module.exports.policies = {
     reLogin: ['hasValidUsername'],
     logout: true,
     status: true,
+    submitEmail: ['hasValidUsername'],
```

Comment:
Let's create a different `hasValidEmail` policy (because emails have different validation than usernames) and use both that and the existing `isLoggedIn`
```suggestion
    submitEmail: ['isLoggedIn', 'hasValidEmail'],
```